gulpfile.js

```diff
@@ -107,6 +128,43 @@
 		});
 	}
 
+	const buildAngular = done => {
+		var processRow = function (row) {
+			var compName = row.source.split("/").pop();
```

Comment:
Could you use [path.basename](https://nodejs.org/api/path.html#path_path_basename_path_ext) here?